### PR TITLE
feat: Add assigned reviewer to comparison table as hidden property

### PR DESCRIPTION
**Pull request by DanielBohme**

### CODE REVIEW

File: src/Components/TagComparisonTable/ColumnDefs/ReviewColumnDefs.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { ColDef } from "@ag-grid-community/core"
+import { getPropertyName } from "../../../utils/helpers"
+
+export const comparisonReviewColumnDefs = (): ColDef[] => [
+    {
+        headerName: "Assigned reviewer",
```
Comment (taustad) [inline src/Components/TagComparisonTable/ColumnDefs/ReviewColumnDefs.tsx:6]:
```suggestion
        headerName: "Assigned reviewers",
```

Comment (DanielBohme) [reply src/Components/TagComparisonTable/ColumnDefs/ReviewColumnDefs.tsx:6]: good catch. will fix

File: src/Components/TagComparisonTable/TagComparisonTable.tsx [hunk 2]
```diff
@@ -69,14 +72,44 @@ function TagComparisonTable({ tags }: Props) {
...
+            } catch (error) {
+                console.error(`Couldn't get tag reviews: ${error}`)
+            }
+        })()
+    }, [])
+
+    const mapTagReviews = (tag: any) => {
+        const reviewers: string[] = []
+        // eslint-disable-next-line array-callback-return
+        const map = tagReviews?.map(async (tagReview: any) => {
```
Comment (taustad) [inline src/Components/TagComparisonTable/TagComparisonTable.tsx:89]: async?

File: src/Components/TagComparisonTable/TagComparisonTable.tsx [hunk 3]
```diff
@@ -69,14 +72,44 @@ function TagComparisonTable({ tags }: Props) {
...
+            if (tagReview.tagNo === tag.tagNo) {
+                for (let i = 0; i < tagReview?.reviewer?.length; i += 1) {
```
Comment (taustad) [inline src/Components/TagComparisonTable/TagComparisonTable.tsx:91]: use forEach?

File: src/Components/TagComparisonTable/TagComparisonTable.tsx (hunk 2 above)
Comment (DanielBohme) [reply src/Components/TagComparisonTable/TagComparisonTable.tsx:89]: didn't notice that one. must have been added automatically when writing as I haven't added async to that. will fix

**Review by taustad — APPROVED** [review]

**Pull request merged**